.github/bot/src/lib/GetFileDiff.ts

```diff
@@ -71,8 +78,9 @@ const getParsedContent = async (
 ): Promise<ParsedContent> => {
   const Github = getOctokit(GITHUB_TOKEN);
   const decodeData = (data: ContentFile) => {
-    const encoding = data.encoding;
-    return Buffer.from(data.content, data.encoding as any).toString();
+    const encoding = data.encoding
+    if (!validateEncoding(encoding)) throw new Error(`Unknown encoding of file ${filename}: ${encoding}`)
```

Comment:
Nit: I have been using assertions for things like this, might be worth moving into the Assertions folder and calling assertEncoding

```
const encoding: EncodingsEnum = assertEncoding(encoding) 
```

I can also do this but would flow better I think + enum

Reply:
`assertEncoding` instead of `validateEncoding` is fine.  If you make such a change, I recommend making it an actual TypeScript assertion:
```ts
function assertEncoding(encoding: string): asserts encoding is Encodings {
	if (!encodings.includes(maybeEncoding as any)) throw new Error(`Unknown encoding of file ${filename}: ${encoding}`)
}
```
This way, TypeScript will know that if that function returns then `encoding` is an `Encodings`.

Reply:
One disadvantage of the assert is that you lose the `fileName` unless you pass it through as a context of some kind.

Reply:
Pushed with this changed to assert.